tox.ini

```diff
@@ -123,3 +124,12 @@ deps =
     parfive[ftp]
 commands =
     python -c "import sunpy; sunpy.self_test()"
+
+
+[testenv:core_deps]
+description = Check the test suite does not fail if only the core dependencies are insatlled.
+deps =
+extras =
+    core
+commands =
+    python -c "import sunpy; sunpy.self_test()"
```

Comment:
Does this actually run the test suite? Wouldn't we need the test extra for that?

Maybe this isn't for this PR, but actually I think having things not in the core extra auto-skip in the test suite might not be a bad idea, so we can actually run the tests here.

Reply:
Well what do you want to happen?

Do we want to be able to run the test suite with some tests that do not error if only the core deps are installed?

We want them to skip tests, raise a message?